Doc/reference/datamodel.rst

```diff
@@ -1526,18 +1526,17 @@ positional arguments; bit ``0x08`` is set if the function uses the
 if the function is a generator. See :ref:`inspect-module-co-flags` for details
 on the semantics of each flags that might be present.
 
-Future feature declarations (``from __future__ import division``) also use bits
```

Comment:
Why remove 'bits'?

Reply:
I tried to wrap line with 80 column, but forget bits in next line.